Source/WebCore/Modules/webauthn/WebAuthenticationUtils.cpp

```diff
@@ -37,6 +37,8 @@
 #include <wtf/text/Base64.h>
 #include <wtf/text/WTFString.h>
 
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
```

Comment:
Can we be more fine grained? It's be good to surround only the code that requires this.